app/helpers/admin/organizations_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+  module OrganizationsHelper
+    def upcoming_conferences_count(organization)
+      organization.conferences.where('end_date >= ?', Date.current).count
+    end
+
+    def past_conferences_count(organization)
+      organization.conferences.count - upcoming_conferences_count(organization)
```

Comment:
Note that this code does TWO counts, the total and the upcoming.
Can we do this in a single query? And create a scope for it?